src/Makefile

```diff
@@ -2,33 +2,20 @@ ELMMAKE=elm-make
   # for locally-modified Elm compiler, create a symlink to something like:
   # /PATH/TO/elm-make/.stack-work/install/x86_64-osx/lts-3.22/7.10.2/bin/elm-make
 
-all: elm-stuff/packages
-	$(ELMMAKE) Main.elm --output ../build/out/sns.js
-	# sed -i '' 's/var Elm = {};/var Elm = {};\nElm["EvalUpdate"] = Elm["EvalUpdate"] || {};\nElm["EvalUpdate"].api = _user$$project$$EvalUpdate$$api;/g' ../build/out/sns.js
+out_from_Native :=aceCodeBox.js outputCanvas.js aceTooltips.js animationLoop.js fileHandler.js deucePopupPanelInfo.js \
+ dotGraph.js colorScheme.js syntaxHighlight.js focus.js keyBlocker.js solverServer.js
+
+out_from_ace-builds :=ace.js mode-little.js mode-elm.js theme-chrome.js
+
+all: prelude examples html
```

Comment:
`prelude` and `examples` are unnecessarily slow for a regular rebuild, which is why they were part of `make clean` only.

Reply:
I always use the `watchmake` script, which default to `make html`, so this doesn't affect me but I don't know what @MikaelMayer and @ravichugh do.

Reply:
I talked to Ravi and he thought it was fine, and said you were on your own branch so it wouldn't affect you. I didn't ping Mikael :P Is there a slack channel or something for discussing SnS? Anyways, I can change the default `make` to just be `make html`, then have a `make everything` or `make all` for `prelude` and `examples`. I think it's counterintuitive for `clean` to build anything

Reply:
Can we create a #sketch-n-sketch channel on the UChicagoCS slack?